python/python/bystro/search/save/handler.py

```diff
@@ -285,80 +327,14 @@ async def go(  # pylint:disable=invalid-name
                 for locus in loci:
                     loci_fh.write(locus + "\n")
 
-            pool = pa.default_memory_pool()
-            with Timer() as timer:
-                dataset = ds.dataset(parent_dosage_matrix_path, format="arrow")
-                mask = pc.field("locus").isin(loci)
-
-                # Use the dataset's scanner to stream batches through a filter
-                scanner = dataset.scanner(
-                    filter=mask,
-                    use_threads=True,
-                    batch_size=SAVE_FILTER_BATCH_READ_SIZE,
```

Comment:
Just to confirm I'm understanding correctly, this section isn't necessary because all of this is handled in the dosage-filter go code now?

Reply:
correct!